cli/test/test_cmd_version.py

```diff
@@ -13,5 +13,5 @@ def test_version(self):
             runner = CliRunner()
             result = runner.invoke(cmd_version.cli)
             self.assertEqual(result.exit_code, 0)
-
+            self.assertEqual(__version__, __popper_version__)
```

Comment:
if we make the default be `0.0.0`, we can add one more assertion that tests that the value of the version is not `0.0.0`.